### ISSUE TITLE
npm run test => error

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint#8522]
I'm trying `npm run test` on the branch `master`, but got an error.

**output:**
Validating rules

  [▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬]

  15794 passing (1m)  
  1 failing

  1) bin/eslint.js reading from stdin gives a detailed error message if no config file is found:
     AssertionError: expected '\nOops! Something went wrong! :(\n\nESLint couldn\'t find the plugin "eslint-plugin-markdown". This can happen for
a couple different reasons:\n\n1. If ESLint is installed globally, then make sure eslint-plugin-markdown is also installed globally. A globally-in
stalled ESLint cannot find a locally-installed plugin.\n\n2. If ESLint is installed locally, then it\'s likely that the plugin isn\'t installed co
rrectly. Try reinstalling by running the following:\n\n    npm i eslint-plugin-markdown@latest --save-dev\n\nIf you still can\'t figure out the pr
oblem, please stop by https://gitter.im/eslint/eslint to chat with the team.\n\n' to match /ESLint couldn't find a configuration file/
      at Function.assert.match (node_modules/chai/lib/chai/interface/assert.js:883:32)
      at getStdout.then.stdout (tests/bin/eslint.js:92:24)
      at process._tickCallback (internal/process/next_tick.js:103:7)

**node version**: v6.9.1
**npm version**: 4.5.0

**Comment by gyandeeps** [comment]
Can you please share your system info and also what kind of CLI did you use since it relates to `stdin`?

**Comment by Aladdin-ADD** [comment]
os: mac os 10.12.2 
terminal: vs code 1.11.0-insider terminal.integrated.shell.linux.
error log(/Users/zsd/.npm/_logs/2017-04-28T14_08_39_558Z-debug.log):
>0 info it worked if it ends with ok
1 verbose cli [ '/Users/zsd/.nvm/versions/node/v6.9.1/bin/node',
1 verbose cli   '/Users/zsd/.nvm/versions/node/v6.9.1/bin/npm',
1 verbose cli   'run',
1 verbose cli   'test' ]
2 info using npm@4.5.0
3 info using node@v6.9.1
4 verbose run-script [ 'pretest', 'test', 'posttest' ]
5 info lifecycle eslint@4.0.0-alpha.1~pretest: eslint@4.0.0-alpha.1
6 silly lifecycle eslint@4.0.0-alpha.1~pretest: no script for pretest, continuing
7 info lifecycle eslint@4.0.0-alpha.1~test: eslint@4.0.0-alpha.1
8 verbose lifecycle eslint@4.0.0-alpha.1~test: unsafe-perm in lifecycle true
9 verbose lifecycle eslint@4.0.0-alpha.1~test: PATH: /Users/zsd/.nvm/versions/node/v6.9.1/lib/node_modules/npm/bin/node-gyp-bin:/Users/zsd/repo/pv/eslint/node_modules/.bin:/Users/zsd/.yarn/bin:/Users/zsd/.nvm/versions/node/v6.9.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
10 verbose lifecycle eslint@4.0.0-alpha.1~test: CWD: /Users/zsd/repo/pv/eslint
11 silly lifecycle eslint@4.0.0-alpha.1~test: Args: [ '-c', 'node Makefile.js test' ]
12 silly lifecycle eslint@4.0.0-alpha.1~test: Returned: code: 1  signal: null
13 info lifecycle eslint@4.0.0-alpha.1~test: Failed to exec test script
14 verbose stack Error: eslint@4.0.0-alpha.1 test: `node Makefile.js test`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/Users/zsd/.nvm/versions/node/v6.9.1/lib/node_modules/npm/lib/utils/lifecycle.js:279:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/Users/zsd/.nvm/versions/node/v6.9.1/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:877:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid eslint@4.0.0-alpha.1
16 verbose cwd /Users/zsd/repo/pv/eslint
17 verbose Darwin 16.3.0
18 verbose argv "/Users/zsd/.nvm/versions/node/v6.9.1/bin/node" "/Users/zsd/.nvm/versions/node/v6.9.1/bin/npm" "run" "test"
19 verbose node v6.9.1
20 verbose npm  v4.5.0
21 error code ELIFECYCLE
22 error errno 1
23 error eslint@4.0.0-alpha.1 test: `node Makefile.js test`
23 error Exit status 1
24 error Failed at the eslint@4.0.0-alpha.1 test script 'node Makefile.js test'.
24 error Make sure you have the latest version of node.js and npm installed.
24 error If you do, this is most likely a problem with the eslint package,
24 error not with npm itself.
24 error Tell the author that this fails on your system:
24 error     node Makefile.js test
24 error You can get information on how to open an issue for this project with:
24 error     npm bugs eslint
24 error Or if that isn't available, you can get their info via:
24 error     npm owner ls eslint
24 error There is likely additional logging output above.
25 verbose exit [ 1, true ]

**Comment by Aladdin-ADD** [comment]
I removed the config `.eslintrc` in home direcory. like this:
```bash
rm -rf ~/.eslintrc
```
and it works.

this problem is very similar to https://github.com/eslint/typescript-eslint-parser/issues/211.
I think eslint locally installed should not use this config, so maybe there is something wrong with eslint. @platinumazure

**Comment by Aladdin-ADD** [comment]
seems Tackling in #8561

**Comment by platinumazure** [comment]
@Aladdin-ADD Is this still an issue? If your issue is resolved, please close at your convenience. Thanks!
